yml/hsc.yaml

```diff
@@ -46,24 +46,84 @@ tables:
     description: Covariance between fiducial ICRS Right Ascension and Declination of centroid.
     mysql:datatype: FLOAT
     fits:tunit: deg^2
-  - name: deblend_incompleteData
-    "@id": "#Object.deblend_incompleteData"
-    datatype: boolean
-    description: One or more bands were not deblended due to an inability to model the PSF.
-    mysql:datatype: BOOLEAN
-    fits:tunit:
   - name: deblend_nChild
     "@id": "#Object.deblend_nChild"
     datatype: int
     description: Number of children this object has (defaults to 0)
     mysql:datatype: INTEGER
     fits:tunit:
+  - name: deblend_nPeaks
+    "@id": "#Object.deblend_nPeaks"
+    datatype: int
+    description: Number of peaks this parent footprint has (even if the deblender failed or skipped this blend)
+    mysql:datatype: INTEGER
+    fits:tunit:
+  - name: deblend_failed
+    "@id": "#Object.deblend_failed"
+    datatype: boolean
+    description: Deblender failed to deblend this source
+    mysql:datatype: BOOLEAN
+    fits:tunit:
   - name: deblend_skipped
     "@id": "#Object.deblend_skipped"
     datatype: boolean
     description: Deblender skipped this source
     mysql:datatype: BOOLEAN
     fits:tunit:
+  - name: deblend_isolatedParent
+    "@id": "#Object.deblend_isolatedParent"
+    datatype: boolean
+    description: Deblender skipped this footprint because there was only a single peak
+    mysql:datatype: BOOLEAN
+    fits:tunit:
+  - name: deblend_parentTooBig
+    "@id": "#Object.deblend_parentTooBig"
+    datatype: boolean
+    description: Deblender skipped this source because the parent footprint was too large.
+    mysql:datatype: BOOLEAN
+    fits:tunit:
+  - name: deblend_tooManyPeaks
+    "@id": "#Object.deblend_tooManyPeaks"
+    datatype: boolean
+    description: Deblender skipped this source because there were too many peaks in the Footprint.
+    mysql:datatype: BOOLEAN
+    fits:tunit:
+  - name: deblend_masked
+    "@id": "#Object.deblend_masked"
+    datatype: boolean
+    description: Deblender skipped this source because there were too many masked pixels.
+    mysql:datatype: BOOLEAN
+    fits:tunit:
+  - name: deblend_incompleteData
+    "@id": "#Object.deblend_incompleteData"
+    datatype: boolean
+    description: One or more bands were not deblended due to an inability to model the PSF.
+    mysql:datatype: BOOLEAN
+    fits:tunit:
+  - name: deblend_iterations
+    "@id": "#Object.deblend_iterations"
+    datatype: int
+    description: Number of iterations during deblending
+    mysql:datatype: INTEGER
+    fits:tunit:
+  - name: deblend_peak_center_x
+    "@id": "#Object.deblend_peak_center_x"
+    datatype: int
+    description: x-coordinate of the peak after source detection
+    mysql:datatype: INTEGER
+    fits:tunit:
```

Comment:
Please supply the unit `pixel` here (assuming I'm understanding this correctly).  Also: I note that this is declared as `int`, i.e., not supporting fractional-pixel positions.  I don't know if that's what was intended, so can you just confirm that this is what you want?

Reply:
Yes, this is intended. These are _not_ the centroids of the objects, they are the integer peak positions from the detection algorithm, which is useful for tracking detections across different versions of the science pipelines because they aren't dependent on measurements, deblending, etc.